src/components/structures/TimelinePanel.tsx

```diff
@@ -967,8 +952,6 @@ class TimelinePanel extends React.Component<IProps, IState> {
 
         if (!this.state.events.includes(ev)) return;
 
-        this.recheckFirstVisibleEventIndex();
-
         // Need to update as we don't display event tiles for events that
         // haven't yet been decrypted. The event will have just been updated
```

Comment:
this comment isn't quite accurate now

Reply:
The comment [pre-dates](https://github.com/matrix-org/matrix-react-sdk/pull/1334/files) the [old pre-join UTD logic](https://github.com/matrix-org/matrix-react-sdk/pull/3881), so I think it's unrelated